Documentation/prerequisites-for-building.md

```diff
@@ -12,12 +12,19 @@ PowerShell also needs to be available from the PATH environment variable (it's t
 
 Install basic dependency packages:
 
+First add a new package source to be able to install clang-3.9:
+```sh
+echo "deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.9 main" | sudo tee /etc/apt/sources.list.d/llvm.list
+wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
+sudo apt-get update
 ```
-sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev liblttng-ust liblttng-ust-dev uuid uuid-dev
```

Comment:
I've updated the list based on a clean Ubuntu 14.04 docker container testing. It is very minimal - just cmake clang-3.9 libicu52 and libunwind8.